packages/scheduled-tasks/README.md

```diff
@@ -0,0 +1,253 @@
+
+**Plugin for <a href="https://github.com/sapphiredev/framework">@sapphire/framework</a> to add support for scheduled tasks.**
+
+[![GitHub](https://img.shields.io/github/license/sapphiredev/plugins)](https://github.com/sapphiredev/plugins/blob/main/LICENSE.md)
+[![codecov](https://codecov.io/gh/sapphiredev/plugins/branch/main/graph/badge.svg?token=QWL8FB16BR)](https://codecov.io/gh/sapphiredev/plugins)
+[![npm bundle size](https://img.shields.io/bundlephobia/min/@sapphire/plugin-scheduled-tasks?logo=webpack&style=flat-square)](https://bundlephobia.com/result?p=@sapphire/plugin-scheduled-tasks)
+[![npm](https://img.shields.io/npm/v/@sapphire/plugin-scheduled-tasks?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/plugin-scheduled-tasks)
+
+</div>
+
+## Description
+
+Many bots have features that need to run periodically, such as uploading analytics data, reminders for users, birthdays, scheduled giveaways, undoing moderation actions, and more. Several implemented solutions exist for this, but as with many time-based processing attempts, they are often flawed and unreliable. This plugin is our solution, enabling you to schedule tasks and save them in services like Redis and SQS with ease.
+
+## Features
+
+-   Full TypeScript support
+-   Includes ESM entrypoint
+
+## Installation
+
+`@sapphire/plugin-scheduled-tasks` depends on the following packages. Be sure to install these along with this package!
+
+-   [`@sapphire/framework`](https://www.npmjs.com/package/@sapphire/framework)
+
+In case you want to use bull as your provider:
+
+-   [`bull`](https://www.npmjs.com/package/bull)
+
+In case you want to use sqs as your provider:
+
+-   [`sqs-consumer`](https://www.npmjs.com/package/sqs-consumer)
+-   [`sqs-producer`](https://www.npmjs.com/package/sqs-producer)
+
+You can use the following command to install this package with bull, or replace `npm install` with your package manager of choice.
+
+```sh
+npm install @sapphire/plugin-scheduled-tasks @sapphire/framework bull
+```
+
+---
+
+## Usage
+
+This registers the necessary options and methods in the Sapphire client to be able to use the schedule plugin.
+
+```typescript
+// Main bot file
+// Be sure to register the plugin before instantiating the client.
+import { ScheduledTaskRedisStrategy } from '@sapphire/plugin-scheduled-tasks/register-redis';
+
+// Or if you want to use sqs
+import { ScheduledTaskSQSStrategy } from '@sapphire/plugin-scheduled-tasks/register-sqs';
+```
+
+Then, you can pass the imported Strategy into the configuration options in your SapphireClient extension class or initializer. This will either be located in your new SapphireClient constructor call, or super in your constructor method if you use an extension class.
+
+```typescript
+{
+	tasks: {
+		strategy: new ScheduledTaskRedisStrategy();
+		//or with sqs
+		strategy: new ScheduledTaskSQSStrategy({
+			/* you can add your SQS options here */
+		});
+	}
```

Comment:
You can't use semicolons inside an object, it's invalid syntax.

Reply:
Also, missing space between `//` and `or`.

Reply:
how the hell did TS not trigger on the semi

Reply:
bah. Prettier added the semi because it didn't detect it as a proper TS object. I had to add `const options = ` to it and then it no longer did so. Good catch for sure.